cmd/cosign/cli/verify/verify_blob.go

```diff
@@ -273,6 +323,12 @@ func base64signature(sigRef string, bundlePath string) (string, error) {
 			return "", err
 		}
 		targetSig = []byte(b.Base64Signature)
+	case rfc3161TimestampPath != "":
```

Comment:
I left a comment above about this, but `rfc3161TimestampPath` should be storing a timestamp, not a signature, correct?

Reply:
Further, a 3161 timestamp should be in addition to the bundle, this makes it either/or.